examples/kotlin/src/main/kotlin/com/example/ondeck/mysql/Models.kt

```diff
@@ -7,12 +7,12 @@ package com.example.ondeck.mysql
 import java.sql.Timestamp
 import java.time.Instant
 
-enum class VenuesStatus(val value: String) {
+enum class VenueStatus(val value: String) {
```

Comment:
This name changed because the table is called `venue` not `venues`. The changed name wasn't taken into account before.